app/views/find/v2/results/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="app-filter-layout">
+  <div class="app-filter-layout__filter">
+    <div class="app-filter__header">
+      <h2 class="govuk-heading-m">Filters</h2>
+
+      <div class="app-filter__content">
+      </div>
+    </div>
+  </div>
+  <div class="app-filter-layout__content">
+    <ul class="app-search-results">
+      <% @results.each do |course| %>
+        <li class="app-search-results__item" data-qa="course">
+          <%= govuk_summary_card(
+            classes: ["course-summary-card"],
+            title: govuk_link_to(
+              find_course_path(
+                provider_code: course.provider_code,
+                course_code: course.course_code
+              ),
+              class: 'govuk-link govuk-!-font-size-24',
+              data: { qa: 'course__link' }
+            ) do
+              content_tag(:span, course.provider.provider_name, class: 'app-search-result__provider-name', data: { qa: 'course__provider_name' }) +
+              content_tag(:span, course.name_and_code, class: 'app-search-result__course-name', data: { qa: 'course__name' })
+            end
+          ) %>
+        </li>
```

Comment:
For now I decided not to create a component just yet.

Until the need arrives ...